### PR TITLE
Update DevOps Checklist

### PR DESCRIPTION
**Pull request by brikis98**
Updates based on customer feedback in [this Slack thread](https://gruntwork-community.slack.com/archives/CHH9Y3Z62/p1590775012223800?thread_ts=1590774430.222100&cid=CHH9Y3Z62).

### CODE REVIEW

File: _data/devops-checklist.yml [hunk 1]
```diff
@@ -139,9 +139,11 @@
...
         Use <a href="https://aws.amazon.com/elasticache/" target="_blank">Elasticache</a> if you want to use Redis or
         Memcached for key-value storage. Use <a href="https://aws.amazon.com/dynamodb/" target="_blank">DynamoDB</a> if
-        you need a managed, eventually consistent document store. If you need other NoSQL databases, such as MongoDB
-        or Couchbase, you'll need to run it yourself (see the
-        <a href="https://gruntwork.io/infrastructure-as-code-library/" target="_blank">Gruntwork Library</a>).
+        you need a managed, eventually consistent key-value store. Use
+        <a href="https://aws.amazon.com/documentdb/" target="_blank">DocumentDB</a> if you need a managed, scalable document store.
+        If you need other NoSQL databases, such as MongoDB, Couchbase, or InfluxDB, you'll need to look to other
+        managed services (e.g., <a href="https://mlab.com/" target="_blank">mLab</a>) or you can run them yourself (see
+        the <a href="https://gruntwork.io/infrastructure-as-code-library/" target="_blank">Gruntwork Library</a>).
```
[the remark this answers is not in the document]
Comment (brikis98) [reply _data/devops-checklist.yml]: Updated: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/304/commits/3001dbdf5147519f427500e387f36a04863aa4d3

**Comment by brikis98** [comment]
Thanks for the review! Merging now. @eak12913 Could you deploy when you get a chance?

**Comment by eak12913** [comment]
This should be out!

**Comment by brikis98** [comment]
Thanks Eug!

**Pull request merged**